### PR TITLE
migrate to Minilla

### PR DESCRIPTION
**Pull request by skaji**
Use of Module::Install for new distributions is discouraged by its maintainers.
https://metacpan.org/pod/Module::Install#WARNING

So why don't you migrate to Minilla?
I hope this change makes Data-MessagePack-Stream more maintainable.

Note:
* I confirmed travis-ci passed with this change.
https://travis-ci.com/skaji/Data-MessagePack-Stream/builds/77454121
* Currently Data-MessagePack-Stream cannot be built on Ubuntu 18.04. See [this gist](https://gist.github.com/skaji/fb60c95603bf71c321319e2aa2359455).
I will create another PR that fixes this.

### CODE REVIEW

File: builder/MyBuilder.pm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+sub ACTION_code {
+    my ($self, @argv) = @_;
+
+    my $spec = $self->_infer_xs_spec(catfile("lib", "Data", "MessagePack", "Stream.xs"));
+    my $archive = catfile("msgpack-1.4.2", "src", ".libs", "libmsgpackc.a");
+    if (!$self->up_to_date($archive, $spec->{lib_file})) {
+        $self->_build_msgpack or die;
+        push @{$self->{properties}{objects}}, $archive; # XXX
```
Comment (skaji) [inline builder/MyBuilder.pm]: I'm not sure this is good, but it *works* :)

File: builder/MyBuilder.pm [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        $file = catfile(split /\//, $file);
+        die "miss $file" if !-e $file;
+        utime $mtime, $mtime, $file or die "utime $mtime, $mtime, $file: $!";
+    }
+}
+
+sub _build_msgpack {
+    my $self = shift;
+
+    $self->_set_mtime;
```
Comment (skaji) [inline builder/MyBuilder.pm]: We should restore original mtime in msgpack-1.4.2.tar.gz; otherwise end-users might be asked for having automake tools.

**Comment by skaji** [comment]
@typester I guess syohex is busy. I would appreciate it if you review this PR too.

**Comment by skaji** [comment]
ping?

**Comment by skaji** [comment]
@typester Thank you for merging this!

FYI: Minilla will take care of almost all releasing stuff (e.g., bump version, rewrite Changes file).
So in order to release Data-MessagePack-Stream, all you have to do is just typing `minil release`.

**Comment by typester** [comment]
Yeah, I know `minil release` stuff though, I use docker environment for releasing perl modules now, so it is a bit difficult.

It is not impossible at all but needs complicated docker command-line (or to create my own image) that I can't remember :'(

**Comment by skaji** [comment]
Fair enough!

**Pull request merged**